Examples/Scripts/Python/propagation.py

```diff
@@ -4,9 +4,7 @@
 import acts
 import acts.examples
 
-from acts.examples import GenericDetector, AlignedDetector
-
-from common import getOpenDataDetectorDirectory
+from acts.examples import GenericDetector
 
```

Comment:
Same here, as an example script I have some commented out alternatives further down in this file to encourage changing the file. If we want to keep that, I would think we should keep these (unused) imports.

Reply:
Thanks, ATLAS habit, there we get flake complaining about unused imports, will brig them back.